extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -71,23 +67,11 @@ class AmpStickyAd extends AMP.BaseElement {
     this.ad_ = children[0];
     this.setAsOwner(this.ad_);
     whenUpgradedToCustomElement(dev().assertElement(this.ad_)).then(ad => {
-      if (!ad.whenBuilt) {
-        // TODO(@zhouyx, #9126): Cleanup once make sure the fix works
-        dev().error(TAG, 'element whenBuilt still do not exist!');
-        timerFor(this.win).delay(() => {
-          ad.whenBuilt().then(() => {
-            this.mutateElement(() => {
-              toggle(this.element, true);
-            });
-          });
-        }, 1000);
-      } else {
-        ad.whenBuilt().then(() => {
-          this.mutateElement(() => {
-            toggle(this.element, true);
-          });
+      ad.whenBuilt().then(() => {
```

Comment:
It's JS style when using Promises (we needed promises because of callback hell). When using promises, instead of chaining a bunch of `#then` calls inside a then block, you return the promise instead.

```js
then(() => {
  return ad.whenBuilt().then(() => {
    // this is callback hell
  });
});

//vs
then(() => {
  return ad.whenBuilt();
}).then(() => {
  // left indent is the same, so callback hell is avoided.
});